File: CS/NetFramework/net48/06_Display/PlaceHolder.cs
using GrapeCity.Win.Editors;
using InputManWin13_Demo._02_Format;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace InputManWin13_Demo._06_Display
{
    public partial class PlaceHolder : InputManWin13_Demo.DemoBase_settings
    {
        public PlaceHolder()
        {
            InitializeComponent();
            InitializeForm();
        }
        private void InitializeForm()
        {
            // xmlファイルからデータ取得
            System.String filePath = Application.StartupPath;
            this.dataSet1.ReadXml(filePath + "\\Resources\\prefecture.xml");

            DataView dv1 = new DataView(dataSet1.Tables[0]);
            gcComboBox1.DataSource = dv1;
            gcComboBox1.SelectedIndex = 0;

            for (int i = 1; i < gcComboBox1.ListColumns.Count; i++)
            {
                gcComboBox1.ListColumns[i].AutoWidth = true;
            }
            gcComboBox2.SelectedIndex = 0;
        }

        private void checkBox1_CheckedChanged(object sender, EventArgs e)
        {
            // PlaceHolderプロパティ
            gcTextBox1.PlaceHolder.Enable = checkBox1.Checked;
            gcTextBox1.PlaceHolder.FloatingStyle.Font = new Font("メイリオ", int.Parse(gcNumber2.Value.ToString()));
            gcMask1.PlaceHolder.Enable = checkBox1.Checked;
            gcMask1.PlaceHolder.FloatingStyle.Font = new Font("メイリオ", int.Parse(gcNumber2.Value.ToString()));
            gcMask1.PlaceHolder.FloatingStyle.ForeColor = Color.Red;
            gcDateTime1.PlaceHolder.Enable = checkBox1.Checked;
            gcDateTime1.PlaceHolder.FloatingStyle.Font = new Font("メイリオ", int.Parse(gcNumber2.Value.ToString()));
            gcDateTime1.PlaceHolder.FloatingStyle.ForeColor = Color.Red;
            gcTime1.PlaceHolder.Enable = checkBox1.Checked;
            gcTime1.PlaceHolder.FloatingStyle.Font = new Font("メイリオ", int.Parse(gcNumber2.Value.ToString()));
            gcTime1.PlaceHolder.FloatingStyle.ForeColor = Color.Red;
            gcDate1.PlaceHolder.Enable = checkBox1.Checked;
            gcDate1.PlaceHolder.FloatingStyle.Font = new Font("メイリオ", int.Parse(gcNumber2.Value.ToString()));
            gcTimeSpan1.PlaceHolder.Enable = checkBox1.Checked;
            gcTimeSpan1.PlaceHolder.FloatingStyle.Font = new Font("メイリオ", int.Parse(gcNumber2.Value.ToString()));
            gcNumber1.PlaceHolder.Enable = checkBox1.Checked;
            gcNumber1.PlaceHolder.FloatingStyle.Font = new Font("メイリオ", int.Parse(gcNumber2.Value.ToString()));
            gcNumber1.PlaceHolder.FloatingStyle.ForeColor = Color.Red;
            gcComboBox1.PlaceHolder.Enable = checkBox1.Checked;
            gcComboBox1.PlaceHolder.FloatingStyle.Font = new Font("メイリオ", int.Parse(gcNumber2.Value.ToString()));
        }

        private void gcNumber2_ValueChanged(object sender, EventArgs e)
        {
            gcTextBox1.PlaceHolder.FloatingStyle.Font = new Font("メイリオ", int.Parse(gcNumber2.Value.ToString()));
            gcMask1.PlaceHolder.FloatingStyle.Font = new Font("メイリオ", int.Parse(gcNumber2.Value.ToString()));
            gcDateTime1.PlaceHolder.FloatingStyle.Font = new Font("メイリオ", int.Parse(gcNumber2.Value.ToString()));
            gcTime1.PlaceHolder.FloatingStyle.Font = new Font("メイリオ", int.Parse(gcNumber2.Value.ToString()));
            gcDate1.PlaceHolder.FloatingStyle.Font = new Font("メイリオ", int.Parse(gcNumber2.Value.ToString()));
            gcTimeSpan1.PlaceHolder.FloatingStyle.Font = new Font("メイリオ", int.Parse(gcNumber2.Value.ToString()));
            gcNumber1.PlaceHolder.FloatingStyle.Font = new Font("メイリオ", int.Parse(gcNumber2.Value.ToString()));
            gcComboBox1.PlaceHolder.FloatingStyle.Font = new Font("メイリオ", int.Parse(gcNumber2.Value.ToString()));
        }

        private void gcComboBox2_SelectedIndexChanged(object sender, EventArgs e)
        {
            // PlaceHolderAlignmentプロパティ
            gcTextBox1.PlaceHolder.FloatingStyle.PlaceHolderAlignment = (GrapeCity.Framework.Views.PlaceHolderAlignment)gcComboBox2.SelectedIndex;
            gcMask1.PlaceHolder.FloatingStyle.PlaceHolderAlignment = (GrapeCity.Framework.Views.PlaceHolderAlignment)gcComboBox2.SelectedIndex;
            gcDateTime1.PlaceHolder.FloatingStyle.PlaceHolderAlignment = (GrapeCity.Framework.Views.PlaceHolderAlignment)gcComboBox2.SelectedIndex;
            gcTime1.PlaceHolder.FloatingStyle.PlaceHolderAlignment = (GrapeCity.Framework.Views.PlaceHolderAlignment)gcComboBox2.SelectedIndex;
            gcDate1.PlaceHolder.FloatingStyle.PlaceHolderAlignment = (GrapeCity.Framework.Views.PlaceHolderAlignment)gcComboBox2.SelectedIndex;
            gcTimeSpan1.PlaceHolder.FloatingStyle.PlaceHolderAlignment = (GrapeCity.Framework.Views.PlaceHolderAlignment)gcComboBox2.SelectedIndex;
            gcNumber1.PlaceHolder.FloatingStyle.PlaceHolderAlignment = (GrapeCity.Framework.Views.PlaceHolderAlignment)gcComboBox2.SelectedIndex;
            gcComboBox1.PlaceHolder.FloatingStyle.PlaceHolderAlignment = (GrapeCity.Framework.Views.PlaceHolderAlignment)gcComboBox2.SelectedIndex;
        }
    }
}

File: VB/NetFramework/net48/06_Display/PlaceHolder.vb
Imports System.Windows.Forms
Imports System.Data
Imports System.Drawing

Namespace _06_Display
    Partial Public Class PlaceHolder
        Inherits InputManWin13_Demo.DemoBase_settings

        Public Sub New()
            InitializeComponent()
            InitializeForm()
        End Sub

        Private Sub InitializeForm()
            ' xmlファイルからデータ取得(GcComboBox)
            Dim filePath As System.String = Application.StartupPath
            Me.dataSet1.ReadXml(filePath & "\Resources\prefecture.xml")

            Dim dv1 As New DataView(dataSet1.Tables(0))
            gcComboBox1.DataSource = dv1
            gcComboBox1.SelectedIndex = 0

            For i As Integer = 1 To gcComboBox1.ListColumns.Count - 1
                gcComboBox1.ListColumns(i).AutoWidth = True
            Next
            gcComboBox2.SelectedIndex = 0
        End Sub

        Private Sub checkBox1_CheckedChanged(sender As Object, e As EventArgs) Handles checkBox1.CheckedChanged
            gcTextBox1.PlaceHolder.Enable = checkBox1.Checked
            gcTextBox1.PlaceHolder.FloatingStyle.Font = New Font("メイリオ", CInt(gcNumber2.Value))
            gcMask1.PlaceHolder.Enable = checkBox1.Checked
            gcMask1.PlaceHolder.FloatingStyle.Font = New Font("メイリオ", CInt(gcNumber2.Value))
            gcMask1.PlaceHolder.FloatingStyle.ForeColor = Color.Red
            gcDateTime1.PlaceHolder.Enable = checkBox1.Checked
            gcDateTime1.PlaceHolder.FloatingStyle.Font = New Font("メイリオ", CInt(gcNumber2.Value))
            gcDateTime1.PlaceHolder.FloatingStyle.ForeColor = Color.Red
            gcTime1.PlaceHolder.Enable = checkBox1.Checked
            gcTime1.PlaceHolder.FloatingStyle.Font = New Font("メイリオ", CInt(gcNumber2.Value))
            gcTime1.PlaceHolder.FloatingStyle.ForeColor = Color.Red
            gcDate1.PlaceHolder.Enable = checkBox1.Checked
            gcDate1.PlaceHolder.FloatingStyle.Font = New Font("メイリオ", CInt(gcNumber2.Value))
            gcTimeSpan1.PlaceHolder.Enable = checkBox1.Checked
            gcTimeSpan1.PlaceHolder.FloatingStyle.Font = New Font("メイリオ", CInt(gcNumber2.Value))
            gcNumber1.PlaceHolder.Enable = checkBox1.Checked
            gcNumber1.PlaceHolder.FloatingStyle.Font = New Font("メイリオ", CInt(gcNumber2.Value))
            gcNumber1.PlaceHolder.FloatingStyle.ForeColor = Color.Red
            gcComboBox1.PlaceHolder.Enable = checkBox1.Checked
            gcComboBox1.PlaceHolder.FloatingStyle.Font = New Font("メイリオ", CInt(gcNumber2.Value))
        End Sub

        Private Sub gcNumber2_ValueChanged(sender As Object, e As EventArgs) Handles gcNumber2.ValueChanged
            gcTextBox1.PlaceHolder.FloatingStyle.Font = New Font("メイリオ", CInt(gcNumber2.Value))
            gcMask1.PlaceHolder.FloatingStyle.Font = New Font("メイリオ", CInt(gcNumber2.Value))
            gcDateTime1.PlaceHolder.FloatingStyle.Font = New Font("メイリオ", CInt(gcNumber2.Value))
            gcTime1.PlaceHolder.FloatingStyle.Font = New Font("メイリオ", CInt(gcNumber2.Value))
            gcDate1.PlaceHolder.FloatingStyle.Font = New Font("メイリオ", CInt(gcNumber2.Value))
            gcTimeSpan1.PlaceHolder.FloatingStyle.Font = New Font("メイリオ", CInt(gcNumber2.Value))
            gcNumber1.PlaceHolder.FloatingStyle.Font = New Font("メイリオ", CInt(gcNumber2.Value))
            gcComboBox1.PlaceHolder.FloatingStyle.Font = New Font("メイリオ", CInt(gcNumber2.Value))
        End Sub

        Private Sub gcComboBox2_SelectedIndexChanged(sender As Object, e As EventArgs) Handles gcComboBox2.SelectedIndexChanged
            ' PlaceHolderAlignmentプロパティ
            gcTextBox1.PlaceHolder.FloatingStyle.PlaceHolderAlignment = CType(gcComboBox2.SelectedIndex, GrapeCity.Framework.Views.PlaceHolderAlignment)
            gcMask1.PlaceHolder.FloatingStyle.PlaceHolderAlignment = CType(gcComboBox2.SelectedIndex, GrapeCity.Framework.Views.PlaceHolderAlignment)
            gcDateTime1.PlaceHolder.FloatingStyle.PlaceHolderAlignment = CType(gcComboBox2.SelectedIndex, GrapeCity.Framework.Views.PlaceHolderAlignment)
            gcTime1.PlaceHolder.FloatingStyle.PlaceHolderAlignment = CType(gcComboBox2.SelectedIndex, GrapeCity.Framework.Views.PlaceHolderAlignment)
            gcDate1.PlaceHolder.FloatingStyle.PlaceHolderAlignment = CType(gcComboBox2.SelectedIndex, GrapeCity.Framework.Views.PlaceHolderAlignment)
            gcTimeSpan1.PlaceHolder.FloatingStyle.PlaceHolderAlignment = CType(gcComboBox2.SelectedIndex, GrapeCity.Framework.Views.PlaceHolderAlignment)
            gcNumber1.PlaceHolder.FloatingStyle.PlaceHolderAlignment = CType(gcComboBox2.SelectedIndex, GrapeCity.Framework.Views.PlaceHolderAlignment)
            gcComboBox1.PlaceHolder.FloatingStyle.PlaceHolderAlignment = CType(gcComboBox2.SelectedIndex, GrapeCity.Framework.Views.PlaceHolderAlignment)
        End Sub
    End Class
End Namespace

